frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/review-tab/advanced-tab.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { VMWizardStorage } from '../../types';
+import { getStorages } from '../../selectors/selectors';
+import { VolumeWrapper } from '../../../../k8s/wrapper/vm/volume-wrapper';
+import { VolumeType } from '../../../../constants/vm/storage';
+import { getBooleanAsEnabledValue } from '../../../../utils/strings';
+
+import './review-tab.scss';
+
+const AdvancedReviewConnected: React.FC<AdvancedReviewConnectedProps> = (props) => {
+  const { storages } = props;
+
+  const cloudInitEnabledValue = getBooleanAsEnabledValue(
+    storages.filter(
```

Comment:
sorry, I forgot we have a better function for this: `iGetCloudInitNoCloudStorage`

Reply:
Done!